jobs/build/build-sync/build.groovy

```diff
@@ -46,14 +46,6 @@ def buildSyncGenInputs() {
     echo("Generating SRC=DEST and ImageStreams for arches")
     def images = imageList ? "--images '${imageList}'" : ''
     def excludeArchesParam = ""
-    def emergencyArgs = ""
-    
-    //EMERGENCY WORKAROUND; REMOVE!
-    if (params.BUILD_VERSION == "4.10") {
-        emergencyArgs = " --emergency-ignore-issues --exclude-arch aarch64 --exclude-arch ppc64le --exclude-arch s390x"
```

Comment:
There still may be some remaining issues.  For the moment, just drop all the `--exclude-arch` parameters and leave the rest in place.

Reply:
Never mind, looks like we're okay now.
/lgtm